packages/online-playground/src/App.vue

```diff
@@ -90,12 +103,10 @@ if (!hash) {
 }
 
 // persist state
-watchEffect(() => {
-  const newHash = store
-    .serialize()
-    .replace(/^#/, useDevMode.value ? `#__DEV__` : `#`)
```

Comment:
I removed this `replace` logic; things still appear to be functional but please double-check

Reply:
Maybe it's now the `__PROD__` that needs to be removed, maybe it's done by the serialize function now. For this kind of things I would look at the vuejs/core playground itself